online-shopping-website/frontend/src/Components/Seller/SellerProductsForm.js

```diff
@@ -87,6 +88,18 @@ export const SellerForm = () => {
         }
     }
 
+    const handleCheckBoxChange = (event) => {
+        console.log(`Before: ${productData.promoted}`)
+        if (event.target.name === "promoted") {
+            setProductData({
+                ...productData,
+                [event.target.name]: event.target.checked
+            });
+        }
+        console.log(`After: ${productData.promoted}`)
```

Comment:
```suggestion
```
If the debugging is unnecessary, let's remove it!

Reply:
just did it lmao